addons/event/models/event_event.py

```diff
@@ -426,6 +426,8 @@ def _compute_event_mail_ids(self):
                 ]
             if command:
                 event.event_mail_ids = command
+                if not event.event_mail_ids:
```

Comment:
It is weird, if you synchronize with a template that has no mail I would keep it void. People may add them afterwards, but we should keep the synchronization mechanism logical.